Objects/Converters/ConverterRhinoGh/ConverterRhinoGhShared/ConverterRhinoGh.Geometry.cs

```diff
@@ -724,22 +724,19 @@ public RH.Mesh MeshToNative(Mesh mesh)
     }
     m.Faces.CullDegenerateFaces();
 
-    return m;
-  }
-
-  private bool HasInvalidMultiplicity(RH.NurbsCurve curve)
```

Comment:
This method had no references, so I removed it

Reply:
Perfect🙌🏼